lib/workers/repository/update/pr/changelog/azure.spec.ts

```diff
@@ -0,0 +1,263 @@
+import * as httpMock from '../../../../../../test/http-mock';
+import * as semverVersioning from '../../../../../modules/versioning/semver';
+import * as hostRules from '../../../../../util/host-rules';
+import type { BranchUpgradeConfig } from '../../../../types';
+import { getChangeLogJSON } from '.';
+
+jest.mock('../../../../../modules/datasource/npm');
```

Comment:
```suggestion
jest.mock('../../../../../modules/datasource');
```

better mock datasource completely